src/app/pages/LiveRadioPage/__snapshots__/index.test.jsx.snap

```diff
@@ -533,6 +533,7 @@ exports[`Radio Page Main should match snapshot for Canonical 1`] = `
 }
 
 .c7 {
+  z-index: 1;
```

Comment:
The snapshot updates have highlighted that the `z-index` styling change in the `bbc-psammead-media-player` impacts across all media pages. It isn't changing depending on the `showLoadingImage` prop. Could this be an issue? My gut says no but just wanted to flag.

Reply:
🤔 not sure, but worth checking the other pages - what does the z-index=1 mean?

Reply:
It's used to explicitly set the layering of components (i.e. rather than just using html hierarchy). By default everything is set to 'auto' - setting it to 1 on the outer iframe means it will appear on top of other elements.

https://www.w3schools.com/cssref/pr_pos_z-index.asp

It's used in the psammead component to make sure that the once the iframe has loaded - it appears on top of the loading image.

Reply:
Screw it even if visually this doesn't change anything it's still nasty. Going to fix in Psammead.

Just give me 2 weeks to get this one-line change in and the package updated 🕺

Reply:
Psammead PR for review 🙏  https://github.com/bbc/psammead/pull/3640

Reply:
Fixed with that psammead pr